litmus-portal/frontend/src/pages/HomePage/index.tsx

```diff
@@ -2,7 +2,7 @@ import { Card, CardActionArea, Typography } from '@material-ui/core';
 import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
 import React from 'react';
 import { useHistory } from 'react-router-dom';
-import CustomInfoFilled from '../../components/InfoFilled/index';
+import CustomInfoFilled from '../../components/CustomInfoFilled/index';
```

Comment:
You don't need the CustomSomething component names...only if the names overlap with a library component use Custom